[CI] XPackRestIT test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version} failing

**Build Scans:**
- [elasticsearch-periodic #5216 / release-tests](https://gradle-enterprise.elastic.co/s/vr73tyvw5ig3y)
- [elasticsearch-pull-request #44738 / release-tests](https://gradle-enterprise.elastic.co/s/2bnyg773eqwda)
- [elasticsearch-periodic #5210 / release-tests](https://gradle-enterprise.elastic.co/s/cajt5w23yi2k6)
- [elasticsearch-pull-request #44655 / release-tests](https://gradle-enterprise.elastic.co/s/vqtqifig74ge4)
- [elasticsearch-periodic #5204 / release-tests](https://gradle-enterprise.elastic.co/s/ubxbtokoqw7q6)
- [elasticsearch-periodic #5174 / release-tests](https://gradle-enterprise.elastic.co/s/z2ui7o76iuzwu)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:yamlRestTest" --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}" -Dtests.seed=E6C22E69350F89EA -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=lag-TZ -Dtests.timezone=Asia/Khandyga -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.test.rest.XPackRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Bp0%3Desql%2F60_usage%2FBasic%20ESQL%20usage%20output%20(telemetry)%20non-snapshot%20version%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [esql/60_usage:166]: field [esql.functions] doesn't have length [123]
Expected: <123>
     but: was <124>
```

**Issue Reasons:**
- [main] 5 consecutive failures in test test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}
- [main] 5 consecutive failures in step release-tests
- [main] 3 consecutive failures in pipeline elasticsearch-periodic
- [main] 2 consecutive failures in pipeline elasticsearch-pull-request
- [main] 6 failures in test test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version} (54.5% fail rate in 11 executions)
- [main] 6 failures in step release-tests (54.5% fail rate in 11 executions)
- [main] 4 failures in pipeline elasticsearch-periodic (66.7% fail rate in 6 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (40.0% fail rate in 5 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -248,9 +248,6 @@ tests:
 - class: org.elasticsearch.index.reindex.ReindexNodeShutdownIT
   method: testReindexWithShutdown
   issue: https://github.com/elastic/elasticsearch/issues/118040
-- class: org.elasticsearch.xpack.test.rest.XPackRestIT
-  method: test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}
-  issue: https://github.com/elastic/elasticsearch/issues/117862
 
 # Examples:
 #

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
@@ -163,4 +163,4 @@ setup:
   - match: {esql.functions.cos: $functions_cos}
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
-  - length: {esql.functions: 123} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 124} # check the "sister" test above for a likely update to the same esql.functions length check

